tests/src/test/resources/META-INF/dirigible/camel_test/calledFromCamel.js

```diff
@@ -0,0 +1,7 @@
+exports.onMessage = (message) => {
```

Comment:
Make this an `.mjs` file and use ESM exports

Reply:
changed to mjs with export function...